features/src/main/feature/camel-features.xml

```diff
@@ -92,10 +92,16 @@
 
     <feature name="jackson" version="${jackson2-version}">
         <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/${jackson2-version}</bundle>
-        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.fasterxml.jackson.core/jackson-core/${jackson2-version}$overwrite=merge</bundle>
         <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/${jackson2-version}</bundle>
     </feature>
 
+    <feature name="jackson" version="2.16.2">
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-databind/2.16.2</bundle>
+        <bundle dependency='true'>wrap:mvn:com.fasterxml.jackson.core/jackson-core/2.16.2$overwrite=merge</bundle>
+        <bundle dependency='true'>mvn:com.fasterxml.jackson.core/jackson-annotations/2.16.2</bundle>
+    </feature>
```

Comment:
No hard-coded version anymore please, add the version to this section https://github.com/apache/camel-karaf/blob/main/pom.xml#L544-L576